plugins/woocommerce/src/Admin/RemoteInboxNotifications/TransformerService.php

```diff
@@ -56,13 +57,25 @@ public static function apply( $target_value, array $transformer_configs, $defaul
 				throw new InvalidArgumentException( "Unable to find a transformer by name: {$transformer_config->use}" );
 			}
 
-			$transformed_value = $transformer->transform( $target_value, $transformer_config->arguments, $default );
-			// if the transformer returns null, then return the previously transformed value.
```

Comment:
@moon0326 Since you originally implemented this, I'd like to get your opinion on this behaviour change. Previously, it would return the previously transformed value, but now, it returns default.

I've looked at all our current payment gateway suggestions specs and it seems to work as intended. Let me know if there's anything I should be concerned about, thanks!